CONTRIBUTING.md

```diff
@@ -10,7 +10,22 @@ We actively welcome your pull requests.
 5. Make sure your code lints.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
 
-## Contributor License Agreement ("CLA")
+### Forking, cloning and running Rapid
+
+- Make sure to use the latest `node` with something like  `nvm install node`
```

Comment:
The use of 'something like' in the contributing guide could be improved for clarity and precision. This phrase may convey uncertainty and give the impression of a guess, which may not be suitable for the guide's purpose. Furthermore, it's important to consider contributors who do not utilize 'nvm' and ensure that our documentation accommodates a broader range of development setups.

Reply:
Indeed. I will add a note here as well, with some options. Thank you!